composer.json

```diff
@@ -21,5 +21,13 @@
     ],
     "require": {
         "php": ">=5.6.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "MoneroPHP\\": "src/"
+        },
+        "psr-0": {
+            "MoneroPHP": "src/"
+        }
```

Comment:
The `psr-0` autoloader is deprecated. It should only set the `psr-4` autoloader.